lib/vagrant-openshift/action/setup_openshift_mirrors.rb

```diff
@@ -0,0 +1,47 @@
+#--
+# Copyright 2014 Red Hat, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#++
+
+module Vagrant
+  module Openshift
+    module Action
+      class SetupOpenShiftMirrors
+        include CommandHelper
+
+        def initialize(app, env, options)
+          @app = app
+          @env = env
+          @options = options
+        end
+
+        def call(env)
+          do_execute(env[:machine], %{
+echo "Setting up 'mirror.openshift.com' for EPEL ..."
+set -e
+rm -f /tmp/epel-mirror.repo /tmp/set-build-image-args.sh
+curl -s  https://mirror.openshift.com/mirror/epel/epel7.repo | sed "s/^\\[epel/[epel-openshift/" > /tmp/epel-mirror.repo
+echo 'export OS_BUILD_IMAGE_ARGS="--mount /etc/yum.repos.d/epel-mirror.repo:/etc/yum.repos.d/epel-mirror.repo"' > /tmp/set-build-image-args.sh
+
+sudo mv /tmp/epel-mirror.repo /etc/yum.repos.d/epel-mirror.repo
+sudo mv /tmp/set-build-image-args.sh /etc/profile.d/set-build-image-args.sh
```

Comment:
Lots of caveats with `/etc/profile.d` -- place global environment in `/etc/environment`